### ISSUE TITLE
auth modal – rendering behind youtube video in ff/ie

### ISSUE THREAD

**Issue by barryclark** [DoSomethingArchive/legacy-website#985]
In Firefox 24.30 and IE 11.0.9600.16518, after the user clicks "sign in", the window for user info entry window is rendered behind the video window, preventing the user from seeing or entering information. The window is obscured and static -cannot be moved out from behind the video window. See attached screen capture.

This is not an issue for Chrome or on Safari.

**Comment by barryclark** [comment]
Trello card - https://trello.com/c/Pb4YQ1VI/8-pop-up-for-user-sign-up-renders-behind-youtube-video-window

**Comment by DFurnes** [comment]
Was able to reproduce issue. Interestingly not an issue in current version of Firefox either, so great catch on TSG's part. Looking into a fix.

**Comment by DFurnes** [comment]
Gonna pick this up in a few days when YouTube embeds are un-removed from the template markup.

**Comment by barryclark** [comment]
@DFurnes is this blocked by another issue? #969?

**Comment by DFurnes** [comment]
I'm not seeing any YouTube videos on the site anymore. Is there anywhere to test this?

**Comment by barryclark** [comment]
@DFurnes You can add a youtube video in the static content template and test there.

**Comment by DFurnes** [comment]
Cool cool. Will look into this today.

**Comment by barryclark** [comment]
:beers:
